snapcraft/pluginhandler.py

```diff
@@ -169,12 +169,15 @@ def _load_code(self, plugin_name, properties, part_schema):
 
         plugin = _get_plugin(module)
         options = _make_options(part_schema, properties, plugin.schema())
+        # For backwards compatibility we add the project options as an option
+        setattr(options, 'project', self._project_options)
```

Comment:
This will need to go to the base plugin and it will break badly unless I add `**kwargs`, `*args`. What is your take?